src/main/webapp/app/exercises/programming/shared/instructions-render/extensions/programming-exercise-task.extension.ts

```diff
@@ -81,7 +78,7 @@ export class ProgrammingExerciseTaskExtensionWrapper implements ArtemisShowdownE
                 componentRef.instance.showTestDetails =
                     (this.exercise.type === ExerciseType.PROGRAMMING && (this.exercise as ProgrammingExercise).showTestNamesToStudents) || false;
 
-                this.appRef.attachView(componentRef.hostView);
+                // this.appRef.attachView(componentRef.hostView);
```

Comment:
If this is no longer needed remove it instead of commenting it out.

Reply:
Jup, no longer needed as the view is already attached to a ViewContainer.